addons/l10n_tr_nilvera_einvoice/data/template/l10n_tr_nilvera_einvoice.account.tax.code-tr.csv

```diff
@@ -0,0 +1,39 @@
+"id","name","name@tr_TR","code","percentage","code_type"
```

Comment:
Weird consistency with the double quotes, imo to be consistent with other csv in odoo all info should be in double quotes